bin/reportdaily.py

```diff
@@ -60,230 +64,1246 @@ class MissingSubCommand(ValueError):
+        team_id_fk = "1"
+
+        # sql command
+        sql_cmd_trainee = f"""
+        INSERT INTO
+        trainee (NAME_TRAINEE, START_YEAR, GRADUATION_YEAR, DATABASE_VERSION,
+                 NUMBER_OF_TEAMS, DURATION, CREATION_DATE,TEAM_ID)
+        VALUES
+        ("{trainee_name}", "{trainee_start_year}" , "{trainee_end_duration}",	"{self.VERSION}",
+         "{trainee_number_teams}","{trainee_duration}","{trainee_current_day}","{team_id_fk}");
+        """
+
+        sql_cmd_team = f"""
+        INSERT INTO
+        team (TEAM_NAME,TEAM_NUMBER,TEAM_START,TEAM_END)
+        VALUES
+        ("{team_name}","{team_number}","{team_start}","{team_end}");
+        """
+
+        # now its time to execute sql command with data and fill the database
+        self._execute_sql(sql_cmd_team)
+        self._execute_sql(sql_cmd_trainee)
+
+    def _adapt_changes_to_database(self):
+        """
+        """
+        # We need to collect all the changed data
+        log.debug(f"""
+        Replace old values in database with the new
+        %s""", self.sql_data)
+        trainee_name = str(self.sql_data.get("name"))
+        # print(trainee_name)
+        trainee_current_day = self.sql_data.get("current_day")
+        trainee_duration = float(self.sql_data.get("duration"))
+        trainee_start_year = self.sql_data.get("start_year")
+        trainee_end_duration = self.sql_data.get("end_duration_education")
+        trainee_number_teams = int(self.sql_data.get("count_teams"))
+        # team data
+        team_name = self.sql_data.get("team")
+        team_number = self.sql_data.get("team_number")
+        # team duration
+        team_start = self.sql_data.get("team_time_start")
+        team_end = self.sql_data.get("team_time_end")
+        team_id_fk = "1"
+        team_pk = "1"
+        trainee_pk = "1"
+
+        # sql command
+        sql_cmd_trainee = f"""
+        REPLACE INTO
+        trainee (TRAINEE_ID,NAME_TRAINEE, START_YEAR, GRADUATION_YEAR, DATABASE_VERSION,
+                 NUMBER_OF_TEAMS, DURATION, CREATION_DATE,TEAM_ID)
+        VALUES
+        ("{trainee_pk}","{trainee_name}", "{trainee_start_year}" , "{trainee_end_duration}",	"{self.VERSION}",
+         "{trainee_number_teams}","{trainee_duration}","{trainee_current_day}","{team_id_fk}");
+        """
+
+        sql_cmd_team = f"""
+        REPLACE INTO
+        team (TEAM_ID,TEAM_NAME,TEAM_NUMBER,TEAM_START,TEAM_END)
+        VALUES
+        ("{team_pk}","{team_name}","{team_number}","{team_start}","{team_end}");
+        """
+
+        # now its time to execute sql command with data and fill the database
+        self._execute_sql(sql_cmd_team)
+        self._execute_sql(sql_cmd_trainee)
+
+        # overwrite sql database with the changes
+
+    def close(self):
+        """
+        Close database manually
+        """
+        log.debug("Database closed")
+        self.connection.close()
+
+def cmd_init(args, configpath, DATABASEPATH):
```

Comment:
You should treat `DATABASEPATH` the same as `configpath`, use lowercase:

```suggestion
def cmd_init(args, configpath, databasepath):
```